ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/SourceDescription.java

```diff
@@ -103,9 +105,11 @@ public SourceDescription(
         readQueries,
         writeQueries,
         dataSource.getSchema().fields().stream().map(
-            field -> new FieldSchemaInfo(
-                field.name(), SchemaUtil.getSchemaFieldName(field))
-            ).collect(Collectors.toList()),
+            field -> {
```

Comment:
why change the formatting here?

Reply:
Don't remember but improved the formatting! :)